### ISSUE TITLE
[Bug]: Veeam.Backup.PowerShell 12 or higher is required to run the Veeam VBR As Built Report.

### ISSUE THREAD

**Issue by aderusha** [AsBuiltReport/AsBuiltReport.Veeam.VBR#187]
### Veeam Backup & Replication version

12

### Bug description

Running from within the Veeam Powershell console, I get the error:

Report generation fails with the error: `New-AsBuiltReport : Veeam.Backup.PowerShell 12 or higher is required to run the Veeam VBR As Built Report. Update the Veeam Backup & Replication console that provide the required modules.`

### Command-line input

New-AsBuiltReport -Report Veeam.VBR -Target server.example.com -Username 'user@domain.com' -Password 'securepass' -Format Text,Html -OutputFolderPath 'C:\Support\AsBuiltReports' -AsBuiltConfigFilePath 'C:\Support\AsBuiltReports\report.json' -ReportConfigFilePath 'C:\Support\AsBuiltReports\AsBuiltReport.Veeam.VBR.json' -Verbose -EnableHealthCheck

### Steps to reproduce

```
Find-Module -Name 'AsBuiltReport.*' -Repository 'PSGallery'
Install-Module -Name 'AsBuiltReport.Core'
Install-Module -Name 'AsBuiltReport.Veeam.VBR'
New-AsBuiltReport -Report Veeam.VBR -Target server.example.com -Username 'user@domain.com' -Password 'securepass' -Format Text,Html -OutputFolderPath 'C:\Support\AsBuiltReports' -AsBuiltConfigFilePath 'C:\Support\AsBuiltReports\report.json' -ReportConfigFilePath 'C:\Support\AsBuiltReports\AsBuiltReport.Veeam.VBR.json' -Verbose -EnableHealthCheck

VERBOSE: Loading As Built Report configuration from 'C:\Support\AsBuiltReports\report.json'.
VERBOSE: Populating RepositorySourceLocation property for module AsBuiltReport.Core.
VERBOSE: Loading module from path 'C:\Program Files\WindowsPowerShell\Modules\AsBuiltReport.Core\1.4.0\AsBuiltReport.Core.psm1'.
VERBOSE: Populating RepositorySourceLocation property for module AsBuiltReport.Veeam.VBR.
VERBOSE: Loading module from path 'C:\Program Files\WindowsPowerShell\Modules\AsBuiltReport.Veeam.VBR\0.8.11\AsBuiltReport.Veeam.VBR.psm1'.
VERBOSE: Loading AsBuiltReport.Veeam.VBR report configuration file from path 'C:\Support\AsBuiltReports\AsBuiltReport.Veeam.VBR.json'.
VERBOSE: Setting report filename to 'Veeam Backup & Replication As Built Report'.
VERBOSE: Populating RepositorySourceLocation property for module AsBuiltReport.Core.
VERBOSE: Loading module from path 'C:\Program Files\WindowsPowerShell\Modules\AsBuiltReport.Core\1.4.0\AsBuiltReport.Core.psm1'.
WARNING: [ 12:40:45:204 ] [  Module  ] - AsBuiltReport.Core 1.4.0 is currently installed.
VERBOSE: [ 12:40:45:768 ] [ Document ] - Document 'Veeam Backup & Replication As Built Report' processing started.
VERBOSE: [ 12:40:45:800 ] [ Document ] - Please wait while the Veeam VBR As Built Report is being generated.
VERBOSE: [ 12:40:45:800 ] [ Document ] - Executing report style script from path 'C:\Program Files\WindowsPowerShell\Modules\AsBuiltReport.Core\1.4.0\AsBuiltReport.Core.Style.ps1'.
VERBOSE: [ 12:40:45:833 ] [ Document ] - Setting global document options.
VERBOSE: [ 12:40:45:836 ] [ Document ] - Enabling section/heading numbering.
[...]Loads of text settings things up[...]
VERBOSE: [ 12:42:32:047 ] [ Document ] - Trying to import Veeam B&R modules.
VERBOSE: [ 12:42:32:062 ] [ Document ] - Identifying Veeam Powershell module version.
VERBOSE: [ 12:42:32:293 ] [ Document ] - Using Veeam Powershell module version 1.0.
New-AsBuiltReport : Veeam.Backup.PowerShell 12 or higher is required to run the Veeam VBR As Built Report. Update the
Veeam Backup & Replication console that provide the required modules.
At line:1 char:1
+ New-AsBuiltReport -Report Veeam.VBR -Target server.example.com  ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,New-AsBuiltReport
```

### Expected behaviour

Report generation happens

### Screenshots

```
PS C:\Support\AsBuiltReports> Get-Module -Name Veeam.Backup.PowerShell -ListAvailable

    Directory: C:\Program Files\Veeam\Backup and Replication\Console

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Manifest   1.0        Veeam.Backup.PowerShell             {Add-VBRBackupCopyJob, Get-VBRBackupCopyJob, Set-VBRBackupCopyJob, Copy-VBRBackupCopyJob...}

PS C:\Support\AsBuiltReports> (Get-Item "C:\Program Files\Veeam\Backup and Replication\Console\Veeam.Backup.PowerShell.dll").VersionInfo.ProductVersion
12.0.0.1420
```

### Operating System

```
systeminfo | Select-String "OS Name", "OS Version", "OS Build"

OS Name:                   Microsoft Windows Server 2016 Standard
OS Version:                10.0.14393 N/A Build 14393
OS Build Type:             Multiprocessor Free
BIOS Version:              VMware, Inc. VMW71.00V.18227214.B64.2106252220, 6/25/2021
```

### PowerShell Version

```
$PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.14393.7426
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.7426
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### PowerShell Modules

```
Name                    Version
----                    -------
AsBuiltReport.Core      1.4.0
AsBuiltReport.Veeam.VBR 0.8.11
PScribo                 0.10.0
```

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://www.asbuiltreport.com/about/contributing/#reporting-issues-and-bugs).

**Comment by rebelinux** [comment]
What is your veeam version?

**Comment by rebelinux** [comment]
Thanks, fix is scheduled for next release (v0.8.12).

If you need to generate the report urgently, you can perform the manual installation of the module.

https://github.com/AsBuiltReport/AsBuiltReport.Veeam.VBR#github
